contracts/rust/src/lib.rs

```diff
@@ -116,6 +116,10 @@ impl NEP4 for NonFungibleTokenBasic {
 
     fn check_access(&self, account_id: AccountId) -> bool {
         let account_hash = env::sha256(account_id.as_bytes());
+        let signer = env::signer_account_id();
+        if signer == account_id {
+            return true;
```

Comment:
`return` is implicit in Rust fyi, so it can just be:
```
if signer == account_id {
  true
}
```
but no biggie

Reply:
changing return true; to true; results in a test that starts to fail, so there's something weird going on in rust. 

changing to true without semicolon does not compile.

Reply:
I am going to file and issue and submit. This smells fishy and deeper investigation might perhaps uncover some underlying bug